index.html

```diff
@@ -63,6 +67,171 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
           </ul>
         </div>
       </section>
+      <section class="grid">
+        <h2 class="grid-title">My Recent Works</h2>
+        <article class="item1">
+          <img src="assets/ImgPlaceholder@2x.png" alt="" />
+          <div class="item1-description">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="button link">
+              <li><a href="#">css</a></li>
+              <li><a href="#">html</a></li>
+              <li><a href="#">bootstrap</a></li>
+
+              <li><a href="#">Ruby</a></li>
+            </ul>
+            <button type="submit" class="button-orange small-mg">
+              See Project
+            </button>
+          </div>
+        </article>
+        <!-- <div class="grid"> -->
```

Comment:
- [ ] [Optional] Please remove commented code. It makes your repo and your project unprofessional. if you still want to check the previous changes you made you can go back to the previous commits and check.